lib/spack/spack/compilers/cce.py

```diff
@@ -52,7 +52,7 @@ class Cce(Compiler):
 
     @classmethod
     def default_version(cls, comp):
-        return get_compiler_version(comp, '-V', r'[Vv]ersion.*(\d+(\.\d+)+)')
+        return get_compiler_version(comp, '-V', r'[Vv]ersion.*?(\d+(\.\d+)+)')
```

Comment:
It's for non-greedy matching. You can play with it [here](https://regex101.com/r/lVWcSy/1) (spack takes only Group 1 as a version).